incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -41,22 +69,19 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(hasSecret(ctx, fooName, "foo-sec")).Should(BeTrue())
 		// "foo-sec" is not propagated to bar because Secret hasn't been configured in HNCConfiguration.
 		Eventually(hasSecret(ctx, barName, "foo-sec")).Should(BeFalse())
```

Comment:
You could take out the "Eventually" here because we don't expect this to _become_ false - we expect it to _stay_ false.

Reply:
Thanks for the suggestion. I changed the test to address your comment below.